### PR TITLE
Setup structure for lottery scheme

### PR DESCRIPTION
**Pull request by curiecrypt**
## Content

This PR includes the setup structure for the simple lottery construction. 

## Pre-submit checklist

- Branch
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
- PR
    - [X] No clippy warnings in the CI
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested
## Issue(s)

Relates to #37

### CODE REVIEW

**Review by tolikzinovyev — CHANGES_REQUESTED** [review]
Could you make a pull request with the main branch as the target? Right now it targets your `curiecrypt/simplelottery` branch.

File: src/simple_lottery/mod.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//! Simple lottery construction for the Telescope ALBA  scheme.
```
Comment (tolikzinovyev) [inline src/simple_lottery/mod.rs:1]: nit
```suggestion
//! Simple lottery construction for the Telescope ALBA scheme.
```

File: src/simple_lottery/setup.rs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//! ALBA Simple lottery setup parameters.
+
+/// Setup output parameters
+#[derive(Debug, Clone)]
+pub struct LotterySetup {
```
Comment (tolikzinovyev) [inline src/simple_lottery/setup.rs:5]: Could we name it `Setup` to be consistent with centralized telescope?

**Review by tolikzinovyev — COMMENTED** [review]
Can you please rebase the branch also?

**Review by tolikzinovyev — APPROVED** [review]
LGTM

**Review by rrtoledo — APPROVED** [review]

**Pull request merged**